frappe/utils/jinja.py

```diff
@@ -87,7 +88,8 @@ def get_allowed_functions_for_jenv():
 		"_": frappe._,
 		"get_shade": get_shade,
 		"scrub": scrub,
-		"guess_mimetype": mimetypes.guess_type
+		"guess_mimetype": mimetypes.guess_type,
+		"dev_server": os.environ.get('DEV_SERVER', False)
```

Comment:
This variable should be in context.py. Here we only expose methods available to jinja.